rasa/server.py

```diff
@@ -719,12 +714,18 @@ async def retrieve_tracker(request: Request, conversation_id: Text) -> HTTPRespo
         """Get a dump of a conversation's tracker including its events."""
         verbosity = event_verbosity_parameter(request, EventVerbosity.AFTER_RESTART)
         until_time = rasa.utils.endpoints.float_arg(request, "until")
-
-        tracker = await app.ctx.agent.processor.fetch_full_tracker_with_initial_session(
```

Comment:
Why was `fetch_full_tracker_with_initial_session` removed? I added this method in order to fix [a bug reported for the HTTP API](https://github.com/RasaHQ/rasa/pull/12139) 🤔 Only concerned that zombie bugs would reappear.

Reply:
it is still there, but there is a paraemeter that allows you not to do it. if this is always done, this creates a new tracker in cases you are fetching a non existant conversation and sometimes you don't want to create a tracker for that

Reply:
Ah yes I see the `start_session` parameter, but in the else branch `get_tracker` is used which in turn uses `tracker_store.get_or_create_tracker` while `fetch_full_tracker` was using `tracker_store.get_or_create_full_tracker` 🤔